docs/native/src/differences.rst

```diff
@@ -1,3 +1,16 @@
+..  _differences:
+
+Differences between |Office iOS| and |wac|
+==========================================
+
+Office for iOS has two WOPI implementation requirements which are not relevant to Office Online:
+ * **Containers**: Instead of using the ecosystem to traverse the WOPI space, Office for iOS uses containers. A container represents a folder on the cloud storage provider, and a Root Container is a top-level folder. 
```

Comment:
Updated this section to "Unlike |wac|, partners need to implement all Container and Bootstrapper operations. You can find more information about these operations in..."